src/sandstorm/sealed.capnp

```diff
@@ -0,0 +1,29 @@
+# Sandstorm - Personal Cloud Sandbox
+# Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@0xfe5c7cde99284f21;
+
+$import "/capnp/c++.capnp".namespace("sandstorm");
+
+using SystemPersistent = import "supervisor.capnp".SystemPersistent;
+
+interface SealedUiView @0x8ccbcb3c38aa1574 {}
```

Comment:
I'm struggling to understand why we need to define two new interfaces for `UiView`. Could there be some more explanatory text about that here? The `Persistent` interface already has a `sealFor` parameter. How is that different form the sealing that is happening here?